swift/Samples/TicTacToe/Sources/Game/ConfirmQuitScreen.swift

```diff
@@ -0,0 +1,109 @@
+//  
+//  ConfirmQuitScreen.swift
+//  Development - SampleTicTacToe
+//
+//  Created by Astha Trivedi on 3/26/20.
+//
+
+import Workflow
+import WorkflowUI
+
+
+struct ConfirmQuitScreen: Screen {
+    
+    let question: String
+    var onQuitTapped: () -> Void = {}
+    var onCancelTapped: () -> Void = {}
+
+    func viewControllerDescription(environment: ViewEnvironment) -> ViewControllerDescription {
+        return ConfirmQuitViewController.description(for: self, environment: environment)
+    }
+}
+
+
+final class ConfirmQuitViewController: ScreenViewController<ConfirmQuitScreen> {
+    
+    private let questionLabel: UILabel = UILabel(frame: .zero)
+    private let confirmButton: UIButton = UIButton(frame: .zero)
+    private let cancelButton: UIButton = UIButton(frame: .zero)
+    private var onQuitTapped: () -> Void = {}
+    private var onCancelTapped: () -> Void = {}
+
+    required init(screen: ConfirmQuitScreen, environment: ViewEnvironment) {
+        super.init(screen: screen, environment: environment)
+        update(with: screen, environment: environment)
```

Comment:
You don't need this anymore. `screenDidChange` will get called right after `init`.